requirements/_core.txt

```diff
@@ -0,0 +1,268 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
```

Comment:
Is _core.txt still needed and my branch was just out of date? I can delete this file.